EIPS/eip-698.md

```diff
@@ -0,0 +1,85 @@
+---
+eip: 698
+title: OPCODE 0x46 BLOCKREWARD
+author: Cody Burns <dontPanic@codywburns.com>
+discussions-to: https://github.com/ethereum/EIPs/issues/698
+status: Draft
+type: Standards Track
+category: Core
+created: 2017-28-08
+---
+
+## Simple Summary
+
+This EIP adds an additional opcode to the EVM which will return a finalized blocks reward value. 
+
+## Abstract
+
+In the EVM, the 0x40 opcodes are reserved for `Block Information`. Currently reserved opcodes are:
+* `0X40 BLOCKHASH`
+* `0X41 COINBASE`
+* `0X42 TIMESTAMP`
+* `0X43 NUMBER`
+* `0X44 DIFFICULTY`
+* `0X45 GASLIMIT`
+
+This EIP would add an additional opcode, `0x46 BLOCKREWARD`, which would return the block reward for any finalized block. The finalized block reward would include the base reward, uncle payments, and gas.   
+
+## Motivation
+
+
+Per EIP-649 ( #669 ) periodic block reward reductions/variance are now planned in the roadmap, however, this EIP is consensus system agnostic and is most useful in decentralized pool operations and for any contract that benefits from knowing a block reward payout(i.e. Merge mined tokens) 
```

Comment:
[EIP-649](https://eips.ethereum.org/EIPS/eip-649) is now Final. You may want to update the text to reflect this.